src/components/Layout/PageContent.tsx

```diff
@@ -1,17 +1,17 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
 import { Typography } from 'antd';
-
 import style from './PageContent.module.scss';
+
 const { Title } = Typography;
 
 type PageContentProps = {
-  title: string | void;
+  title: string | ReactNode;
```

Comment:
A ReactNode is `type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined`
is it really what is needed here? or juste a `ReactChild` or a `ReactElement`?